examples/custom_http_client.rs

```diff
@@ -0,0 +1,30 @@
+use std::time::Duration;
+
+use hyper_util::client::legacy::connect::HttpConnector;
+use hyper_util::client::legacy::Client as HyperClient;
+use hyper_util::rt::TokioExecutor;
+
+use clickhouse::{error::Result, Client};
+
+#[tokio::main]
+async fn main() -> Result<()> {
+    let connector = HttpConnector::new(); // or HttpsConnectorBuilder
```

Comment:
I think we should note users that it's unstable feature and will be broken 100% eventually